client/blocks/conversations/list.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+/***
+ * Internal dependencies
+ */
+import PostComment from 'blocks/comments/post-comment';
+import { getPostCommentsTree } from 'state/comments/selectors';
+
+export class ConversationCommentList extends React.Component {
+	static propTypes = {
+		blogId: PropTypes.number.isRequired,
+		postId: PropTypes.number.isRequired,
+		commentIds: PropTypes.array.isRequired,
+		post: PropTypes.object, // required by PostComment
+	};
+
+	render() {
+		const { commentIds, commentsTree, post } = this.props;
+		if ( ! commentIds ) {
+			return null;
+		}
+
+		return (
+			<ul className="conversations__comment-list">
+				{ map( commentIds, commentId => {
+					return (
+						<PostComment
+							commentsTree={ commentsTree }
+							key={ commentId }
```

Comment:
Yep - there's actually a note to that effect in `PostComment` from the original developer I believe :)